CHANGELOG.md

```diff
@@ -0,0 +1,65 @@
+# Changelog
+
+Darts is still in an early development phase and we cannot always guarantee backwards compatibility. Changes suceptible to **break existing code** working on a previous release are marked with a "&#x1F534;". Sorry for the inconvenience!
```

Comment:
```suggestion
Darts is still in an early development phase and we cannot always guarantee backwards compatibility. Changes suceptible to **breaking existing code** working on a previous release are marked with a "&#x1F534;". Sorry for the inconvenience!
```
?

Reply:
Also I'd remove the "Sorry for the inconvenience!"